mapper/pojo-base/src/main/java/org/hibernate/search/mapper/pojo/bridge/binding/FilterBindingContext.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.mapper.pojo.bridge.binding;
+
+import java.util.Map;
+import org.hibernate.search.engine.environment.bean.BeanHolder;
+import org.hibernate.search.mapper.pojo.bridge.mapping.programmatic.FilterBinder;
+
+/**
+ * The context provided to the {@link FilterBinder#bind(org.hibernate.search.mapper.pojo.bridge.binding.FilterBindingContext) } method.
+ *
+ * @param <T> The type of factory on the POJO side of the factory.
+ */
+public interface FilterBindingContext<T> extends BindingContext {
+
+	void factory(T factory, Map<String, Object> params);
```

Comment:
As explained in my comment on `@FullTextFilter`, please remove `params` from here.